README.md

```diff
@@ -6,17 +6,21 @@
 import logging
 from gces import EventSubscriber
 
+logger = logging.getLogger(__name__)
 
 TOPIC_NAME = 'super_service_1.events'
```

Comment:
Apenas uma sugestão se for possível... sei que o nome 'super_service_1.events' já estava ali mas seria ótimo ser explícito aqui. Também percebi que já tem o 'super_service_2.events'. Isso me lembra componentes do "Delphi" empilhados nos forms. :smile:

Reply:
Boa!